mender-auth/ipc/server.hpp

```diff
@@ -98,6 +99,7 @@ class Caching {
 	http::Client client_;
 	http::ServerConfig server_config_;
 	http::Server server_;
+	string default_identity_script_path;
```

Comment:
You probably meant to use this in the logic somewhere, but I can't see that it is..?